.travis.yml

```diff
@@ -14,47 +14,28 @@ matrix:
       sudo: true
 env:
   global:
-    - CC_TEST_REPORTER_ID=$TRAVIS_CODE_CLIMATE_TOKEN
+    - CC_TEST_REPORTER_ID=$TRAVIS_CODE_CLIMATE_TOKEN SENDGRID_API_KEY=SGAPIKEY
```

Comment:
_$SGAPIKEY_?

Reply:
hm I was just trying to set a dummy value. Since it's not an actual env var I'm don't think the $ is needed?

Reply:
Ah, nevermind,